File: P1-Linguagem_de_Programação_I/Roteiros de Aula - Java/roteiro-de-aula03.java
import java.util.Scanner;

class roteirofor {
    public static void main(String[] args) {
        System.out.println("\nAdivinhe o número com 'while' em Java!");
        Scanner scanner = new Scanner(System.in);
        int numero_secreto = 7;
        int numero_digitado = 0;

        while (numero_digitado != numero_secreto) {
            System.out.print("Digite um número: ");
            numero_digitado = scanner.nextInt();
            if (numero_digitado != numero_secreto) {
                System.out.println("Número incorreto. Tente novamente.");
            }
        }

        System.out.println("Parabéns! Você acertou!");
        scanner.close();
    }
}

File: P1-Linguagem_de_Programação_I/Roteiros de Aula - Python/roteiro-de-aula03.py
print("\nAdvinhe o número com 'while' em Python!")
num_secreto = 7
num_digitado = 0

while num_digitado != num_secreto:
    num_digitado = int(input("Digite um número: "))
    if num_digitado != num_secreto:
        print("Número incorreto. Tente novamente.")

print("Parabéns! Você acertou!")